games/snake.py

```diff
@@ -1,22 +1,26 @@
 import random
 import curses
 
+from collections import namedtuple
+
+Point = namedtuple('Point', ['x_pos', 'y_pos'])
+
 
 def initial_snake(width, height):
     snake_x = width // 4
     snake_y = height // 2
     return [
-            [snake_y, snake_x],
-            [snake_y, snake_x-1],
-            [snake_y, snake_x-2]
-            ]
+        Point(x_pos=snake_x, y_pos=snake_y),
+        Point(x_pos=snake_x - 1, y_pos=snake_y),
+        Point(x_pos=snake_x - 2, y_pos=snake_y),
+    ]
 
 
 def initial_food(width, height):
-    return [height//2, width//2]
+    return [height // 2, width // 2]
 
 
-class Snake:
+class Snake(object):
```

Comment:
I think the class Snake(object) is a python2 thing. In python 3, it kinda assumes the same. I intend for the project to only support python3